script/templates/README.md

```diff
@@ -39,12 +41,29 @@ setwd('path_to_unzipped_directory')
 install.packages('%1$s', repos=NULL, type='SOURCE')
 ```
 
+### An example
+
+The following code will print a list of all modules available in this library,
+return information about one of them (`example_module`), and then run that
+module:
+```
+library(BioCro)
+library(%1$s)
```

Comment:
The files in `script/templates` are templates used when setting up a module library R package. The setup script (`script/module_library_setup.R`) replaces all the instances of `%1$s` in these files with the user-supplied package name, and moves them to the appropriate location. The final locations are described in `script/template/template_table`.

In the particular case of `script/templates/README.md`, this file will get placed in the root directory of the repository. Then when the repository is viewed through the GitHub web interface, this new `README.md` will take precedence over the one in `docs`. (`docs/README.md` is a readme for the skeleton library.) It will include specific examples that use the name of the package, so they will be easier to follow than writing something like "You can evaluate the example module in your new library by calling `evaluate_module('pkgName:example_module', list(A = 1, B = 2))`, where `pkgName` is the name of the package you specified when calling the setup script."